### PR TITLE
Feature/patient report popup

### PR DESCRIPTION
**Pull request by achen2401**
address story, TN-105:  https://jira.movember.com/browse/TN-105

create modal popup for most recent patient report(s) in patients list

### CODE REVIEW

File: portal/templates/patients_by_org.html [hunk 1]
```diff
@@ -102,6 +102,76 @@ <h4 class="tnth-headline">Patient List</h4>
...
+                  <div id="patientReportContent"></div>
+                  <div id="patientReportMessage" class="error-message"></div>
+              </div>
+              <div class="modal-footer" >
+                  <button type="button" class="btn btn-default btn-consent-close" data-org="{{org}}" data-dismiss="modal" aria-label="Close">{{ _("Close") }}</button>
+              </div>
+          </div>
+      </div>
+  </div>
+  <script>
```
Comment (ivan-c) [inline portal/templates/patients_by_org.html]: Would you be able to move this to `main.js` or another JS file?

Moving JS from dynamically generated HTML into javascript files can improve caching and *eventually* allow us to block inline JS, closing a lot of security vulnerabilities.

No big deal if this isn't feasible at the moment :)

File: portal/templates/profile_macros.html [hunk 2]
```diff
@@ -872,7 +872,7 @@ <h4 class="modal-title">{{ _("Consent to share information") }}</h4>
                         item["filename"] = escapeHtml(item["filename"]);
                         item["document_type"] = escapeHtml(item["document_type"]);
                         item["uploaded_at"] = tnthDates.formatDateString(item["uploaded_at"], "iso");
-                        item["actions"] = '<a title="Download" href="' + '/api/user/' + String(item["user_id"]) + '/user_documents/' + String(item["id"]) + '"><i class="fa fa-arrow-down"></i></a>';
+                        item["actions"] = '<a title="Download" href="' + '/api/user/' + String(item["user_id"]) + '/user_documents/' + String(item["id"]) + '"><i class="fa fa-download"></i></a>';
```
Comment (ivan-c) [inline portal/templates/profile_macros.html]: Could you tag "Download" with `trans` / `__()` / `i18next.t()`?

Can you use `url_for()` here?

Comment (achen2401) [reply portal/templates/profile_macros.html]: ivan, started to make the fix to change to url_for but then I realized the user id and the document id come from the client side - making it infeasible in this case

File: portal/templates/patients_by_org.html (hunk 1 above)
Comment (achen2401) [reply portal/templates/patients_by_org.html]: ivan, thank u for the suggestion, I have moved the block to admin.js.

File: portal/static/js/admin.js [hunk 3]
```diff
@@ -421,7 +421,51 @@ AdminTool.prototype.handleDownloadModal = function() {
...
+              content += "<TH>"+ i18next.t("Report Name") + "</TH><TH>" + i18next.t("Generated (GMT)") + "</TH><TH>" + i18next.t("Downloaded") + "</TH>";
+              documents.forEach(function(item) {
+                  /*
+                   * only draw the most recent, same report won't be displayed
+                   */
+                  if (!existingItems[item["contributor"]]) {
+                    content += "<tr>" +
+                              "<td>" + item["filename"] + "</td>" +
+                              "<td>" + tnthDates.formatDateString(item["uploaded_at"], "iso") + "</td>" +
+                              "<td class='text-center'>" + '<a title="Download" href="' + '/api/user/' + String(item["user_id"]) + '/user_documents/' + String(item["id"])+ '"><i class="fa fa-download"></i></a>' + "</td>"
```
Comment (ivan-c) [inline portal/static/js/admin.js]: Found the one string - "Download"! ;)

Comment (achen2401) [reply portal/static/js/admin.js]: @ivan-c Ivan, thank you! I added the tag and added another column to the report table.  Let me know if you see any more issue.

**Pull request merged**